app/components/layout.js

```diff
@@ -42,7 +42,11 @@ class Layout extends React.Component {
         <Navbar {...this.props} handleConnect={this.connect} handleDisconnect={this.disconnect}/>
         <div className="container">
           <div className="row tools-wrapper">
-            <Toolbelt {...this.props} handshake={this.handshake} handleCpuSampling={this.cpuProfiling}/>
+            {
```

Comment:
Can this be moved to a function that has a normal `if else`? I'm not too sure about a multiline ternary operator